src/status_im/ui/screens/db.cljs

```diff
@@ -60,6 +60,8 @@
 (spec/def ::webview-bridge (spec/nilable any?))
 (spec/def ::status-module-initialized? (spec/nilable boolean?))
 (spec/def ::status-node-started? (spec/nilable boolean?))
+(spec/def ::app-initialized? (spec/nilable boolean?))
+(spec/def ::jail-initialized? (spec/nilable boolean?))
```

Comment:
we don't need this anymore?

Reply:
@flexsurfer Good catch. Removed that too.